src/components/Board/Output/Output.container.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import { connect } from 'react-redux';
+import keycode from 'keycode';
+
+import {
+  cancelSpeech,
+  speak
+} from '../../../providers/SpeechProvider/SpeechProvider.actions';
+
+import { changeOutput } from '../Board.actions';
+import SymbolOutput from './SymbolOutput';
+
+function translateOutput(output, intl) {
+  const translatedOutput = output.map(value => {
+    let translatedValue = { ...value };
+
+    if (value.labelKey) {
+      translatedValue.label = intl.formatMessage({ id: value.labelKey });
+    }
+    return translatedValue;
+  });
+  return translatedOutput;
+}
+
+export class OutputContainer extends Component {
+  static propTypes = {
+    /**
+     * @ignore
+     */
+    intl: intlShape,
+    /**
+     * Array of symbols
+     */
+    output: PropTypes.arrayOf(
+      PropTypes.shape({
+        /**
+         * Image to display
+         */
+        image: PropTypes.string,
+        /**
+         * Label to display
+         */
+        label: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
+      })
+    )
+  };
+
+  static getDerivedStateFromProps(props, state) {
```

Comment:
What about:
```
static getDerivedStateFromProps({ output, intl }, { translatedOutput }) {
```

Reply:
I think it's worse to spread the object in this case.
I like function definitions to explicitly state from where the object is coming `props` or `state`. the spreaded object to me is less readable

Reply:
here for example i thought it was a good fit:
```
        <Scroll {...other}>
          {symbols.map(({ image, label }, index) => (
            <div className="SymbolOutput__value" key={index}>
              <Symbol image={image} label={label} />
            </div>
          ))}
        </Scroll>
```